js/index.d.ts

```diff
@@ -1698,10 +1698,10 @@ export namespace api {
         constructor(properties?: api.IFileEvent);
 
         /** FileEvent file. */
-        public file?: (api.IFile|null);
+        public file?: (api.File|null);
```

Comment:
One worth checking is that these are actually instances of the class not just the interface.